website/docs/guides/migration/versions/06-upgrading-to-v1.2.md

```diff
@@ -13,14 +13,24 @@ title: "Upgrading to v1.2 (prerelease)"
 
 There are no breaking changes for end users of dbt. We are committed to providing backwards compatibility for all versions 1.x. If you encounter an error upon upgrading, please let us know by [opening an issue](https://github.com/dbt-labs/dbt-core/issues/new).
 
+### For consumers of dbt artifacts (metadata)
+
+The manifest schema version has been updated to `v6`. The relevant changes are:
+- Change to `config` default, which includes a new `grants` property with default value `{}`
+- Addition of a `metrics` property, to any node which could reference metrics using the `metric()` function
+
+For users of [state-based selection](understanding-state): This release also includes new logic declaring forwards compatibility for older manifest versions. While running dbt Coree v1.2, it should be possible to use `state:modified --state ...` selection against a manifest produced by dbt Core v1.0 or v1.1.
+
 ## For maintainers of adapter plugins
 
 See GitHub discussion [dbt-labs/dbt-core#5468](https://github.com/dbt-labs/dbt-core/discussions/5468) for detailed information
 
 ## New and changed documentation
 
 - **[Grants](/reference/resource-configs/grants)**: You should now manage access to the datasets you're producing with dbt by using grants instead of using hooks.  If you already use post-hook to apply simple grants, moving to the grants feature will allow you to [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) up your duplicated or boilerplate code.
```

Comment:
@jtcohen6 should we frame this like the others "Resource configs now include [Grants](..."

Reply:
I do feel there's a distinction — **grants** is a totally new feature, and new docs page, whereas the other bullets are describing extensions of / additions to existing pages.

I can definitely take a second pass here